### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by amartya-rs**
### files to check: [review me only](https://github.com/amartya-rs/hhQuiz/pull/1/commits/adf81bcde48dbcfc1888051303601b2426b2c4fc)

### CODE REVIEW

**Review by shraddha15-creator — APPROVED** [review]
Your categories are not visible in the preview link UI. Check once
Rest UI looks good.
Good color theme

File: src/components/top-navigation/TopNav.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Link } from "react-router-dom";
+import "./top-nav.css";
+//import { useAuth } from "../../context/auth-context";
```
Comment (pradipgharge) [inline src/components/top-navigation/TopNav.jsx]: remove commented code

File: src/context/data-context.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+const DataProvider = ({ children }) => {
+   var firebaseConfig = {
+      // The value of `databaseURL` depends on the location of the database
+      databaseURL:
+         "https://hhquiz-4ef5c-default-rtdb.asia-southeast1.firebasedatabase.app",
+   };
+
+   const app = initializeApp(firebaseConfig);
+
+   //fetching quiz data and categories from firebase realtime database
```
Comment (pradipgharge) [inline src/context/data-context.js:16]: remove commented code here as well

**Review by pradipgharge — APPROVED** [review]
code looks clean just remove the commented code, rest looks good to merge.

File: src/components/top-navigation/TopNav.jsx (hunk 1 above)
Comment (amartya-rs) [reply src/components/top-navigation/TopNav.jsx]: Done

File: src/context/data-context.js (hunk 2 above)
Comment (amartya-rs) [reply src/context/data-context.js:16]: Kept this comment for better understanding of the code.

**Pull request merged**